src/main/java/net/atos/zac/templates/model/Template.java

```diff
@@ -0,0 +1,74 @@
+package net.atos.zac.templates.model;
```

Comment:
I think it's better to make the package name and/or class more meaningful. The word `template` is too generic. It's not just any template. It's a SmartDocumentsTemplate right? The word SmartDcouments should be in here I think.

Reply:
I would actually add `SmartDocuments` to both the package and the class name. better to be very explicit in my opinion. Makes finding code a lot easier.